rust/otap-dataflow/crates/otap/src/mock.rs

```diff
@@ -81,7 +81,8 @@ impl ArrowLogsService for ArrowLogsServiceMock {
             while let Ok(Some(batch)) = input_stream.message().await {
                 // Process batch and send status, break on client disconnection
                 let batch_id = batch.batch_id;
-                let status_result = match sender_clone.send(OTAPData::ArrowLogs(batch)).await {
+                let status_result = match sender_clone.send(OTAPData::ArrowLogs(batch).into()).await
```

Comment:
`OtapPdata` and `OTAPData` are very close syntactically, which could make things confusing. I suggest finding a slightly different name for `OTAPData`.